src/main/java/org/symphonyoss/symphony/messageml/elements/Dialog.java

```diff
@@ -0,0 +1,118 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import static org.apache.commons.lang3.StringUtils.containsWhitespace;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+import org.symphonyoss.symphony.messageml.MessageMLContext;
+import org.symphonyoss.symphony.messageml.MessageMLParser;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.util.XmlPrintStream;
+import org.w3c.dom.Node;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Dialog extends Element {
```

Comment:
We started documenting the elements in Javadoc as in TimezonePicker
(because we don't really have a proper specification for elements, beside confluence)
maybe it could be a good thing to keep